### ISSUE TITLE
Error：computing log_prob at site 'y'

### ISSUE THREAD

**Issue by ly15681910119** [ucals/cvae#1]
When the program trains the CVAE, the program reports an error:

                                                                             computing log_prob at site 'y'

The main problem is this line of code:

                                                                  loss = svi.step(inputs, outputs) / inputs.size(0)

**Comment by LXXXXR** [comment]
Same here....

**Comment by xlnn** [comment]
> When the program trains the CVAE, the program reports an error:
> 
> ```
>                                                                          computing log_prob at site 'y'
> ```
> 
> The main problem is this line of code:
> 
> ```
>                                                               loss = svi.step(inputs, outputs) / inputs.size(0)
> ```

me too! A same problems with you!

**Comment by ucals** [comment]
Hey guys... this smells like a compatibility issue.
Pyro is in active development.
When I created this example as a contribution to them, I used Pyro's 1.4.x; Pyro is now over 1.8.
Let me investigate and come back to you..
Cheers!

**Comment by xlnn** [comment]
> Hey guys... this smells like a compatibility issue. Pyro is in active development. When I created this example as a contribution to them, I used Pyro's 1.4.x; Pyro is now over 1.8. Let me investigate and come back to you.. Cheers!

Thank you!

**Comment by shoyua** [comment]
I have the same problem, is there a solution now?

**Comment by shoyua** [comment]
> I have the same problem, is there a solution now?

set  `validate_args=False` in `pyro.sample('y', dist.Bernoulli(mask_loc, validate_args=False).to_event(1), obs=mask_ys)`.  It seems work.
